### ISSUE TITLE
Using vips_sobel() and vips_canny() edge detectors from command prompt

### ISSUE THREAD

**Issue by audovoice** [jcupitt/libvips#1069]
I see in the change log that starting in 8.6 canny edge detection is supported.
I am trying to use 8.7-rc1, vips-dev-w64-web-8.7.0.zip.
I am using it from console on windows10. 
What is the command for using it?

**Comment by audovoice** [comment]
Thanks for the quick response but it does not work yet.
I am running windows if that matters. I installed from the binary.
D:\tiles 2\11>vips.exe canny 00001.png x.v
vips.exe: unknown action "canny"

D:\tiles 2\11>vips.exe --vips-version
libvips 8.7.0-Sun Jan 28 16:38:45 UTC 2018

I am greatly looking forward to the next version. I just found libvips a few weeks ago and it has been very useful making and manipulating tiles for mosaics and also deeplearning.
